### PR TITLE
WX-1385 Remove SAS tokens from TES input log printouts

### PR DESCRIPTION
**Pull request by aednichols**

### CODE REVIEW

File: supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesTask.scala [hunk 1]
```diff
@@ -367,7 +367,15 @@ final case class Input(name: Option[String],
                        path: String,
                        `type`: Option[String],
                        content: Option[String]
-)
+) {
+  override def toString: String = {
+    import akka.http.scaladsl.model.Uri
+
+    // Remove query that may contain SAS token
+    val cleanUrl = url map { Uri(_).copy(rawQueryString = None).toString() }
```
Comment (jgainerdewar) [inline supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesTask.scala:375]: What do you think about only stripping out query params when we can tell the URL contains a SAS token?

Also, can we replace with `?REDACTED` or similar so we can tell something was removed? I can imagine users or our future selves not realizing this is happening and thinking Cromwell is incorrectly stripping the query params before passing the URL to TES.

Comment (aednichols) [reply supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesTask.scala]: I was able to enhance and adopt an existing library function for this!

**Comment by aednichols** [comment]
scalafmt

**Review by THWiseman — APPROVED** [review]

**Comment by aednichols** [comment]
DBMS issues seem to be a systemic issue with a vendor fix in progress here:

https://github.com/testcontainers/testcontainers-java/pull/8131

**Pull request merged**